### PR TITLE
comply with FinSetsForCAP v2022.11-03

### PR DESCRIPTION
**Pull request by mohamed-barakat**
https://github.com/homalg-project/FinSetsForCAP/pull/148

### CODE REVIEW

**Comment by mohamed-barakat** [comment]
@zickgraf: Unfortunately we lost the 16s -> 2s improvement

**Comment by zickgraf** [comment]
This is probably due to the (too) specific logic template
```
CapJitAddLogicTemplate(
    rec(
        variable_names := [ "func" ],
        src_template := "Sum( [ 1, 2, 3, 4 ], func )",
        dst_template := "func( 1 ) + func( 2 ) + func( 3 ) + func( 4 )",
    )
);
```
which has to be adapted.

File: gap/precompiled_categories/FinQuiversAsCCCPrecompiled.gi [hunk 1]
```diff
@@ -811,7 +811,7 @@ end
     if IsBound( cat!.precompiled_functions_added ) then
         
         # COVERAGE_IGNORE_NEXT_LINE
-        #Error( "precompiled functions have already been added before" );
+        Error( "precompiled functions have already been added before" );
```
Comment (zickgraf) [inline gap/precompiled_categories/FinQuiversAsCCCPrecompiled.gi]: If you simply remove this change, the CI should pass.

Comment (mohamed-barakat) [reply gap/precompiled_categories/FinQuiversAsCCCPrecompiled.gi]: Thanks, done.

**Comment by mohamed-barakat** [comment]
> @zickgraf: Unfortunately we lost the 16s -> 2s improvement

This comment is now obsolete.

**Pull request merged**